core/spec/models/spree/order_inventory_spec.rb

```diff
@@ -190,8 +190,8 @@
 
       it 'only attempts to destroy as many units as are eligible, and return amount destroyed' do
         allow(shipment).to receive_messages(inventory_units_for_item: [
-          mock_model(Spree::InventoryUnit, :variant_id => variant.id, :state => 'shipped'),
-          mock_model(Spree::InventoryUnit, :variant_id => variant.id, :state => 'on_hand')
+          mock_model(Spree::InventoryUnit, variant_id: variant.id, state: 'shipped', shipped?: false),
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.